src/nm2/src/nm2_ipv6_address.c

```diff
@@ -113,6 +147,24 @@ void callback_IPv6_Address(
             return;
     }
 
+    /*
+     * (Askey) 
+     * Determine if the only change(s) was valid_lifetime and/or 
+     * preferred_lifetime, if so then skip nm2_ipv6_address_update() that 
+     * follows to avoid triggering the full-blown address provision and 
+     * disruption (can go as much as 6-10 seconds) on LAN side.
+     * To be aware, the skipping of nm2_ipv6_address_update() might come with 
+     * the side effect of not having correct stateful address lifetime on LAN 
+     * side, with the benefit of not causing LAN side disruption.
```

Comment:
```suggestion
     * Check the changes other than valid_lifetime and/or preferred_lifetime.
     * If there were no other changes, skip nm2_ipv6_address_update() that 
     * follows to avoid triggering a full-blown address provisioning, which
     * causes a disruption (as much as 6-10 seconds) on LAN side.
     *
     * Note that skipping of nm2_ipv6_address_update() might come with 
     * the side effect of not having a correct stateful address lifetime on
     * the LAN side, with the benefit of not causing a LAN side disruption.
```